modules/lfs/utils.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
The file name `utils.go` is not a good name.

Reply:
I've changed it to lfs_pointers.go